### ISSUE TITLE
Numerous warnings when building

### ISSUE THREAD

**Issue by wiw-evan-olcott** [GoogleCloudPlatform/recaptcha-enterprise-mobile-sdk#33]
**Describe the bug**

When building an app with this Swift package installed, multiple warnings appear regarding missing paths when GenerateDSYMFile executes.

**Integration Method**

Select the method used to integrate with reCAPTCHA Mobile.

* [X] SPM (iOS)
* [ ] Cocoapods (iOS)
* [ ] Direct Download (iOS)
* [ ] GMaven (Android)

**SDK Version (e.g. 18.0.1):** 18.0.3

**To Reproduce**

1. Add the SPM to an Xcode project (preferably on a machine that has not been used for development of this package)
2. Build
3. Observe multiple warnings regarding missing paths

```
warning: /private/var/netboot/workspace/394528483693365d58e060dcb6d81b86ba17a82c3d5ea8f4c0ca86171eee43d6_01002a48/google3/blaze-out/ios-x86_64-min12.0-applebin_ios-ios_x86_64-opt-ST-c9c798c8be62/bin/third_party/apple_frameworks/WebKit.swift.pcm: No such file or directory
note: while processing /private/var/netboot/workspace/394528483693365d58e060dcb6d81b86ba17a82c3d5ea8f4c0ca86171eee43d6_01002a48/google3/blaze-out/ios-x86_64-min12.0-applebin_ios-ios_x86_64-opt-ST-c9c798c8be62/bin/third_party/apple_frameworks/WebKit.swift.pcm
note: Linking a static library that was built with -gmodules, but the module cache was not found.  Redistributable static libraries should never be built with module debugging enabled.  The debug experience will be degraded due to incomplete debug information.
```
The above is repeated many times with slightly different paths.

**Expected behavior**

No warnings should appear when building the project

**Xcode version for iOS (please complete the following information):**
 - Version [e.g. Xcode 13.4.1]: 14.2

**Additional context**

These paths causing the warnings are all prefixed with `/private/var/netboot/workspace/...`, and are hard-coded into the static library as found in the xcframework. These paths need to be stripped or removed when generating the static libraries for this package.

I believe this note in the warning contains the solution:
```
note: Linking a static library that was built with -gmodules, but the module cache was not found.  Redistributable static libraries should never be built with module debugging enabled.  The debug experience will be degraded due to incomplete debug information.
```

**Comment by mcorner** [comment]
We have seen these in the past and have determined that they are harmless.

To confirm though, these are all warnings and not errors?

**Comment by evan-olcott** [comment]
Yes, these are warnings, not errors. But it's just not good "framework etiquette" to add warnings to other projects - they're not harmless. They make it hard to find ACTUAL warnings if we have to work around the ones we can't control.

Our company doesn't ship builds to production with warnings, and I plead for someone to clean them up so we can implement reCaptcha in our app.

**Comment by mcorner** [comment]
Yep, I hear you and would like to fix it if possible.  Someone looked at this two years ago and determined it was harmless.  I believe it is related to the CLANG_ENABLE_MODULE_DEBUGGING=YES default, but ....I actually don't see this warning myself to reproduce it and fix. So I am guessing there is a difference in flags.  

Do you have a minimal repro of the warning?

**Comment by mcorner** [comment]
Ah, got it.  Happens on archive.  We will look into it.

**Comment by mcorner** [comment]
We believe that we have fixed this internally and should be able to include it in the next release of the SDK.

We will continue to test the internally in the meantime.

Thanks for the report.

**Comment by mcorner** [comment]
We haven't forgotten about this, we did fix it and it will be included in an iOS release that should occur very shortly.

**Comment by mcorner** [comment]
This has been addressed in 18.1.1 released yesterday (no release notes yet)
